docs/design-docs/benchmarks.md

```diff
@@ -0,0 +1,262 @@
+---
+id: benchmarks
+---
+
+# Benchmarks Design
+
+## Goals
+
+The design of `benchmarks` has the following goals to achieve:
+
+**High Code Quality**
+* Extract the common code into the base class, reduce the efforts to maintain different benchmarks.
```

Comment:
Extract the common code into the base class, and reduce the efforts to maintain different benchmarks.

Reply:
Done